docker/dotcms/ROOT/srv/OVERRIDE/WEB-INF/web.xml

```diff
@@ -0,0 +1,617 @@
+<?xml version="1.0"?>
```

Comment:
I think we discussed adding some changes to web.xml for a temporary basis, but already I cannot work out easily what is different in here to the base file in dotCMS/src/main/webapp/WEB-INF/web.xml.   We are going to setting ourselves up for more confusion and pain with differences we may miss.    

Is it just the addition of the parametized values like <param-value>${CMS_HSTS_ENABLED}</param-value>.   If this works can we not change the base web.xml.   

Note in the new base maven server.xml dotCMS/src/main/docker/original/ROOT/srv/OVERRIDE/tomcat/conf/server.xml. we have integrated the configurable parameters that were in the override files and then separately defaulted in 00-config-defaults.sh.   We do not need to separate these out as we can set the default value in place e.g. I am not sure if we can use the same thing in the web.xml also.   It would be good to keep the configurable logic for web.xml in just the one file 

<Executor
            name="tomcatConnectorThreadPool"
            namePrefix="ConnectorThreadPool-"
            maxThreads="${CMS_CONNECTOR_THREADS:-600}"
            minSpareThreads="25"
        />

Reply:
> Is it just the addition of the parameterized values like ${CMS_HSTS_ENABLED}. If this works can we not change the base web.xml.

Yes, the only changes to web.xml are these parameterized values. We also set the defaults for these params in 00-config-defaults.sh.

If we can refactor this PR to provide the same functionality by modifying `dotCMS/src/main/webapp/WEB-INF/web.xml` instead of using having to duplicate that file as we do with server.xml, all the better.

I realize this "shell template" mechanism is deprecated, but this is an urgent need. The current options are untenable.

Reply:
I will test to see if web.xml will fetch parameters from environment variables.